Common/src/config_structure.cpp

```diff
@@ -2904,9 +2905,11 @@ void CConfig::SetPostprocessing(unsigned short val_software, unsigned short val_
   }
 
 #ifndef HAVE_TECIO
-  if (Output_FileFormat == TECPLOT_BINARY) {
-    cout << "Tecplot binary file requested but SU2 was built without TecIO support." << "\n";
-    Output_FileFormat = TECPLOT;
+  for (unsigned short iOutputFile = 0; iOutputFile < nVolumeOutputFiles; iOutputFile++) {
+    if (VolumeOutputFiles[iOutputFile] == TECPLOT_BINARY) {
```

Comment:
This probably won't catch `SURFACE_TECPLOT_BINARY ` for `VolumeOutputFiles[2]` (which is also .szplt format and should be catched right?). But I am already about to push the same thing 
bf2f6ef  to various_fixes #798 together with an SU2_MPI::Error to hinder this implicit conversion to TECPLOT(_ASCII).

Reply:
Aok, thanks, @TobiKattmann. I will not touch this and go with your fix.